libs/common-utils/src/getIsNativeToken.ts

```diff
@@ -24,5 +24,8 @@ export function getIsNativeToken(chainIdOrTokenParams: SupportedChainId | Curren
 
   const nativeToken = NATIVE_CURRENCIES[chainId as SupportedChainId]
 
+  // When token is from Bridge, it's not in the list of native tokens
+  if (!nativeToken) return false
+
```

Comment:
I don't understand how not having `nativeToken` has anything to do with bridge.
Wouldn't it return false already in the check below?

Reply:
We may call `getIsNativeToken` function with a token from for example BSC.
We don't have values for BSC in `NATIVE_CURRENCIES` so we must handle that case.

Reply:
>Wouldn't it return false already in the check below?

It will crash, becase in `doesTokenMatchSymbolOrAddress` we do `token.address` when `token` is undefined.